README.md

```diff
@@ -76,11 +81,19 @@ the actual driver's name.
           args:
```

Comment:
@jsafrane you missed updating the image registry here :-P

Reply:
I can tackle this in a separate PR.